src/main/java/core/basesyntax/service/ReaderService.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.util.List;
+
+public interface ReaderService {
+    List<String> readFromFile(String fromFileName);
```

Comment:
Consider adding a method description comment to explain the purpose of this method.